index.html

```diff
@@ -396,6 +396,36 @@ <h2 class="next-event-header info-note highlighted-text">
           You can check out all our events below!!
         </h2>
 
+        <section class="section-cta" id="cta">
+          <div class="container">
```

Comment:
Is there a need of an extra div with class container, since there is no CSS written for that class, can you remove the div and just keep the `<section>` as the parent container.